templates/splash/splashpage.md

```diff
@@ -4,11 +4,11 @@ title: Canada.ca
 lang: en
 description-en: The Government of Canada website is a single point of access to all programs, services, departments, ministries and organizations of the Government of Canada.
 description-fr: Le site Web du gouvernement du Canada fournit un point d'accès complet à tous les programmes, services, départements, ministères et organismes du gouvernement du Canada.
-creator-en: Government of Canada, Service Canada, Citizen Service Branch, Integrated Channel Management, Web Strategies and Product Management
-creator-fr: Gouvernement du Canada, Service Canada, Direction générale de service aux citoyens, Gestion intégrée des modes de service, Gestion des stratégies et produits Web
-subject-en: Government of Canada, services
-subject-fr: Gouvernement du Canada, services
-dateModified: 2020-06-03
+creator-en: Service Canada
+creator-fr: Service Canada
```

Comment:
Change it for "Government of Canada", this splash page is not for Service Canada specifically.